src/arch/xtensa/drivers/idc.c

```diff
@@ -288,7 +285,7 @@ static enum task_state idc_do_cmd(void *data)
 		  idc_read(IPC_IDCTFC(initiator), core) | IPC_IDCTFC_BUSY);
 
 	/* enable BUSY interrupt */
-	idc_write(IPC_IDCCTL, core, idc->busy_bit_mask);
+	idc_write(IPC_IDCCTL, core, idc->busy_bit_mask | idc->done_bit_mask);
```

Comment:
Why does it set done bit now?

Reply:
Previously slave cores couldn't send messages, so done bitmask could be ignored. Now we need to keep its proper value.